2025-03-09 16:18:18.18 | info | main.go:35 | ------------------------------ # STARTING APPLICATION # ------------------------------ | 
2025-03-09 16:18:18.18 | info | main.go:39 | Server running on 0.0.0.0:5050  | 
2025-03-09 16:18:18.18 | info | main.go:40 | Database connecting to 127.0.0.1:5432 | 
2025-03-09 16:18:18.18 | debug | main.go:50 | ------------------------------- # CONNECT TO DATABASE # ------------------------------ | 
2025-03-09 16:18:18.21 | info | main.go:56 | Successfully Connected to the Database. | Database =eventy
2025-03-09 16:18:18.23 | info | main.go:68 | Tables Created successfully. | 
2025-03-09 16:18:18.23 | debug | routes.go:28 | --------------------------  START ROUTING  ---------------------- | 
2025-03-09 16:18:18.28 | info | main.go:79 | -------------------------------- # Server running on 0.0.0.0:5050 # ------------------------------ | 
2025-03-09 16:18:19.84 | error | user_backoffice.go:246 | Error occured | error=error updating user with ID 2: ERROR: syntax error at or near "+=" (SQLSTATE=42601)
